gozo-stores/src/context/product-context.js

```diff
@@ -0,0 +1,29 @@
+import { createContext, useContext, useEffect, useState } from "react";
+import axios from "axios";
+
+const ProductContext = createContext();
+
+function ProductProvider({children}) {
+
+    const [products, setProducts] = useState([])
+
+    const [loading, setLoading] = useState(true)
+
+    const getProducts = async () => {
+        const res = await axios.get("/api/products")
```

Comment:
Is there any reason to put `products` in context?

Reply:
I thought it will make the code cleaner, nothing else